pandas/tests/series/methods/test_replace.py

```diff
@@ -208,6 +208,14 @@ def test_replace_with_dict_with_bool_keys(self):
         expected = pd.Series(["yes", False, "yes"])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize('dtype', ['int8', 'int16', 'int32', 'int64'])
+    def test_replace_int_with_na(self, dtype):
+        result = pd.Series([0, None]).astype(dtype)
```

Comment:
can you put the issue number as a comment here? see some other tests for an example